eppo_client/models.py

```diff
@@ -96,7 +96,7 @@ class BanditModelData(SdkBaseModel):
 
 class BanditData(SdkBaseModel):
     bandit_key: str
-    model_name: str
+    bandit_model_name: str
```

Comment:
You will need to provide an `alias` for these fields so that Pydantic knows what property names to use when deserializing from JSON ([example payload](https://github.com/Eppo-exp/sdk-test-data/blob/282589ae9cd5f1382c0f9470618399bade934bc7/ufc/bandit-models-v1.json#L9))

Reply:
https://docs.pydantic.dev/latest/api/fields/#pydantic.fields.Field